addons/l10n_ar/demo/respinsc_demo.xml

```diff
@@ -36,6 +36,13 @@
         <value model="res.company" eval="obj().env.ref('l10n_ar.company_ri')"/>
     </function>
 
+    <!-- Products created before the company don't get the default tax set, by default, 
```

Comment:
You can't rely on demo data for the tests, shouldn't this be in the setupclass of the common test class for AR?

Reply:
I think it's the right place for the fix.

the error I'm fixing was spotted by the l10n build, and is reproducible at the installation of l10n_ar with demo data
```
2024-08-22 12:13:59,329 1586869 INFO 17.0-ar-default-taxes-aoû-22 odoo.modules.loading: loading l10n_ar/demo/account_customer_invoice_demo.xml 
2024-08-22 12:14:01,605 1586869 WARNING 17.0-ar-default-taxes-aoû-22 odoo.modules.loading: Module l10n_ar demo data failed to install, installed without demo data 
Traceback (most recent call last):
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 556, in _tag_root
    f(rec)
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 280, in _tag_function
    _eval_xml(self, rec, env)
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 205, in _eval_xml
    return odoo.api.call_kw(model, method_name, args, kwargs)
  File "/home/odoo/dev/odoo/odoo/api.py", line 468, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/home/odoo/dev/odoo/odoo/api.py", line 453, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/odoo/dev/enterprise/account_reports/models/account_move.py", line 61, in action_post
    super().action_post()
  File "/home/odoo/dev/enterprise/account_accountant/models/account_move.py", line 77, in action_post
    res = super().action_post()
  File "/home/odoo/dev/odoo/addons/account/models/account_move.py", line 4105, in action_post
    other_moves._post(soft=False)
  File "/home/odoo/dev/odoo/addons/l10n_ar/models/account_move.py", line 220, in _post
    ar_invoices._check_argentinean_invoice_taxes()
  File "/home/odoo/dev/odoo/addons/l10n_ar/models/account_move.py", line 144, in _check_argentinean_invoice_taxes
    raise UserError(_('There should be a single tax from the "VAT" tax group per line, add it to %r. If you already have it, please check the tax configuration, in advanced options, in the corresponding field "Tax Group".', line.name))
odoo.exceptions.UserError: There should be a single tax from the "VAT" tax group per line, add it to 'Virtual Home Staging'. If you already have it, please check the tax configuration, in advanced options, in the corresponding field "Tax Group".

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/odoo/dev/odoo/odoo/modules/loading.py", line 89, in load_demo
    load_data(env(su=True), idref, mode, kind='demo', package=package)
  File "/home/odoo/dev/odoo/odoo/modules/loading.py", line 71, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 627, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 693, in convert_xml_import
    obj.parse(doc.getroot())
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 613, in parse
    self._tag_root(de)
  File "/home/odoo/dev/odoo/odoo/tools/convert.py", line 569, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing /home/odoo/dev/odoo/addons/l10n_ar/demo/account_customer_invoice_demo.xml:274, somewhere inside
<function model="account.move" name="action_post">
        <value eval="[ref('demo_invoice_1'), ref('demo_invoice_2'), ref('demo_invoice_3'), ref('demo_invoice_4'), ref('demo_invoice_5'), ref('demo_invoice_6'), ref('demo_invoice_7'), ref('demo_invoice_8'), ref('demo_invoice_10'), ref('demo_invoice_11'), ref('demo_invoice_12'), ref('demo_invoice_13'), ref('demo_invoice_14'), ref('demo_invoice_15'), ref('demo_invoice_16'), ref('demo_invoice_17'), ref('demo_invoice_18'), ref('demo_invoice_19')]"/>
    </function>

```
So apparently in l10n_ar is a constraint in l10n_ar that checks there is a tax on all posted invoices, which wasn't the case anymore for the demo data of the module since my fix.